src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -10,7 +10,7 @@ import io.iohk.ethereum.vm.MockWorldState.PS
 
 class OpCodeGasSpec extends FunSuite with OpCodeTesting with Matchers with PropertyChecks {
 
-  override val config = EvmConfig.PostEIP160ConfigBuilder(None)
+  override val config = EvmConfig.PostEIP161ConfigBuilder(None)
```

Comment:
Not sure if this a good idea. After all our priority is ETC. These tests were designed before hard-fork/EIP support was added and they don't offer much support for those :(

I would say they should test the latest in-effect ETC hard-fork. What do you think?
Unless maybe you can think of a clean way to test for both ETC and ETH?

Reply:
Or maybe we could add an EIP-161 specific opcode test? I.e. move these changes to a separate spec, and leave these tests unmodified, WDYT?

Reply:
You definitely have valid point here. I think in the scope of this pr most appropriate would be to move it to other specs like `OpCodeGasSpecPostEIP161`, an additional bonus of that would be that we will see what op codes has been modified in this eip changes.